chain/client/src/view_client.rs

```diff
@@ -774,7 +784,14 @@ impl Handler<NetworkViewClientMessages> for ViewClientActor {
                 }
             }
             NetworkViewClientMessages::TxStatus { tx_hash, signer_account_id } => {
-                if let Ok(Some(result)) = self.get_tx_status(tx_hash, signer_account_id) {
+                if let Ok(Some(result)) = self.get_tx_status(tx_hash, signer_account_id, false) {
```

Comment:
Does this mean that the routed message won't reply to TxStatus with the receipts?

P.S. `get_tx_status(blah-blah, false)` is not helpful. To avoid such APIs, I suggest we introduce independent methods, e.g. `get_tx_status_without_receipts()` + `get_tx_status_with_receipts()`, they may share a **private** `get_tx_status(... bool)` method if necessary.

Reply:
I would agree with you. However the plan is to deprecate routing for rpc requests so I don't think it makes sense to do what you suggested